src/controller/shop.controller.ts

```diff
@@ -71,6 +71,14 @@ export default class ShopController extends BaseController {
     this.success(res, '--shop/deleted', 'shop deleted', 200, null);
   }
 
+  async getAllShops(req: Request, res: Response) {
+    const shops = await prisma.shop.findMany();
+    if (shops.length > 0) {
+      this.success(res, 'All shops', 'Shops have been listed successfully', 200, shops);
+    } else {
+      this.error(res, '--shops-notfound', 'Shops not found', 404);
```

Comment:
sending error isn't ok as this operation isn't an error, it only means the user has no shop, so pls change all this..